app/models/patient.rb

```diff
@@ -580,7 +580,7 @@ def send_assessment(force = false)
       PatientMailer.assessment_sms_weblink(self).deliver_later if ADMIN_OPTIONS['enable_sms'] && !Rails.env.test?
     elsif preferred_contact_method&.downcase == 'telephone call' && responder.id == id
       PatientMailer.assessment_voice(self).deliver_later if ADMIN_OPTIONS['enable_voice'] && !Rails.env.test?
-    elsif ADMIN_OPTIONS['enable_email'] && responder.id == id && !email.blank?
+    elsif preferred_contact_method&.downcase == 'e-mailed web link' && ADMIN_OPTIONS['enable_email'] && responder.id == id && !email.blank?
```

Comment:
Maybe change `!email.blank?` to be `email.present?` as that is what is used elsewhere in this file.

Reply:
all set